packages/flutter_tools/lib/src/commands/test.dart

```diff
@@ -156,7 +156,6 @@ class TestCommand extends FlutterCommand {
 
   @override
   Future<FlutterCommandResult> runCommand() async {
-    await globals.cache.updateAll(await requiredArtifacts);
```

Comment:
Unless I'm totally missing something, this did a second required artifacts update check.  When I stepped through it it called `updateAll` with the same artifacts twice.

Reply:
odd...

I'm sure that is my fault